config/locales/en.yml

```diff
@@ -1733,6 +1733,7 @@ two_factor_authentication.recaptcha.google_policy_link: Privacy Policy
 two_factor_authentication.recaptcha.google_tos_link: Terms of Service
 two_factor_authentication.recaptcha.login_tos_link: Mobile Terms of Use
 two_factor_authentication.recommended: Recommended
+two_factor_authentication.sign_in.recaptcha.disclosure_statement_html: This site is protected by reCAPTCHA and the Google %{google_policy_link_html} and %{google_tos_link_html} apply.
```

Comment:
Actually, one suggestion, let's not use "two_factor_authentication" in this key. The usage has nothing to do with 2FA.